admin/team.md

```diff
@@ -0,0 +1,55 @@
+# The Impastas
+
+## Welcome to our Team!
+
+    Check out our cool video!
+    https://github.com/cse110-fa21-group24/cse110-fa21-group24.git
```

Comment:
Is the link here supposed to take you directly to our team vid or is it just supposed to link to our repo?